packages/legend-studio/style/components/editor/uml-editor/_class-editor.scss

```diff
@@ -93,6 +93,12 @@
     margin-right: 0.5rem;
     width: calc(100% - 36rem);
     min-width: calc(100% - 36rem);
+
+    &__duplicate {
```

Comment:
this is a BEM modifier, we should do `--duplicated` instead. Read more here http://getbem.com/naming/

Reply:
Also, as mentioned before, could we add a little `TimesCircle` icon at the end of `<input>` tag. Also, it's not bad to use `input-group` (See `input.scss`) to have an error message below `<input>`.